docs/sdk/api/sdkMiddlewareHttp.md

```diff
@@ -79,6 +79,36 @@ const client = createClient({
 })
 ```
 
+## `abortController` | `getAbortController`
+
+This is used to signal the retry module to retry the request in an event of a request timeout or service outage.
+
+#### Usage example
+
+```js
+// Use default options
+const httpMiddleware = createHttpMiddleware({
+  host: testHost,
+  timeout: 1000, // time out after 1s
+  fetch,
+  abortController: new AbortController(),
+})
+```
+
+Note however the slight difference in usage of the `getAbortController` property of the http middleware.
+
+```js
+// Use default options
+const httpMiddleware = createHttpMiddleware({
+  host: testHost,
+  timeout: 1000, // time out after 1s
+  fetch,
+  getAbortController: () => new AbortController(),
+})
+```
+
+The is intentional as to ensure a new instance of the AbortController is always created and is independent of each other. Unlike the former (abortController) which only creates a single abortController instance for the middleware, in this very case, if a single request times out, it will propagate to all other http requests that is using the `AbortController` instance. This is useful when a bunch of sent out requests needs to timeout if at least one within the bunch times out.
```

Comment:
I guess you meant to write` js` and not 'is'  ;) so please write it as a code block

Reply:
Thanks @barbara79 do you mean the part that says:

The is intentional as to ensure... ?

Reply:
yes

Reply:
I just saw your change now it looks good :D